.eslintrc.json

```diff
@@ -2,7 +2,8 @@
   "extends": [
     "plugin:@wordpress/eslint-plugin/recommended-with-formatting",
     "plugin:jest/recommended",
-    "plugin:sitekit/main"
+    "plugin:sitekit/main",
+    "plugin:react-hooks/recommended"
```

Comment:
The IB says to only enable one of `react-hook`'s two rules, *but* enabling `react-hooks/rules-of-hooks` didn't generate a single warning (Eugene was worried here => https://github.com/google/site-kit-wp/issues/3072#issuecomment-817684043 )

`react-hooks/rules-of-hooks` gives great warnings for people who are learning hooks